toytools/collect.py

```diff
@@ -0,0 +1,122 @@
+"""Functions to load toyzero image dataset"""
+
+import os
+import re
+
+from typing import List, Tuple, Optional, Set
+import numpy as np
+
+DIR_FAKE = 'fake'
+DIR_REAL = 'real'
```

Comment:
Module level globals can become a problem.

Imagine if we want to load data from "real/" and "real2/" subdirectories.

Refactoring from module-level data and functions to a `class` would be one solution.

Just a comment, no need to fix now.

Reply:
> Imagine if we want to load data from "real/" and "real2/" subdirectories

I think this usecase is rather rare, and 99% of time the default directory layout will be fine. Therefore, it is easier to solve this case by using filesystem tools (e.g. symlinks).